alternator/executor.cc

```diff
@@ -1198,7 +1198,7 @@ rmw_operation::returnvalues rmw_operation::parse_returnvalues(const rjson::value
 
 rmw_operation::rmw_operation(service::storage_proxy& proxy, rjson::value&& request)
     : _request(std::move(request))
-    , _schema(get_table(proxy, _request))
+    , _schema(executor::get_table(proxy, _request))
```

Comment:
I made a number of file static methods type static instead. To be able to have code in a different file. Because executor.cc is huge and takes forever to compile, and makes my intellisense barf, and also separation is nice. 

Could make non-static also, but private static keeps it from being outside accessible.

Reply:
Could also make sense to move such utility functions to a separate source file and header file (we already have one, serialization.hh, but it's a bit of a mess too.... At some point we need to reorganize the Alternator source code, and we have an issue for that, but it probably shouldn't be mixed in a series which changes functionality?)